mono/metadata/unity-utils.c

```diff
@@ -976,6 +976,23 @@ MONO_API int mono_unity_gc_is_disabled()
 #endif	
 }
 
+// Logging
+static UnityLogErrorCallback editorLoggingCallback;
+MONO_API void mono_unity_set_editor_logging_callback(UnityLogErrorCallback callback)
```

Comment:
Are you only going to do this for editor then and not runtime?

Reply:
Yes this is going to be editor only. These errors have always wound up in the logfile but this is the kind of error that would be nice if it were visible in the editor console.